src/set/set.service.ts

```diff
@@ -46,6 +49,13 @@ export class SetService {
                 .populate<{ createdBy: User }>('createdBy')
                 .lean();
         } catch (error) {
+            if (error.code === 11000) {
```

Comment:
just to make sure: Did you make sure that these error messages cannot lead to security leaks and confidential data getting out?

Reply:
No but this is indeed an issue since the slug is no longer unique and the error won´t be thrown.

Removed